src/screens/ChannelListScreen/index.tsx

```diff
@@ -25,6 +26,9 @@ const ChannelListScreen = ({route}) => {
   const isFocused = navigation.isFocused();
   const {channels, goToChatScreen, goToPostDetailScreen, goToCommunityScreen, goToContactScreen} =
     useSignedChannelListScreenHook();
+  const ref = React.useRef(null);
+
+  useScrollToTop(ref);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of `useScrollToTop` hook is correct. However, it would be better to add a type to the ref for better TypeScript support.
```diff
- const ref = React.useRef(null);
+ const ref = React.useRef<FlatList | null>(null);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->